formula-android/src/main/java/com/instacart/formula/android/internal/ActivityManager.kt

```diff
@@ -61,9 +55,8 @@ internal class ActivityManager<Activity : FragmentActivity>(
         delegate.attachActivity(activity)
         delegate.onLifecycleStateChanged(Lifecycle.State.CREATED)
         val renderView = fragmentRenderView ?: throw callOnPreCreateException(activity)
-        uiSubscription = fragmentState.subscribe {
-            Utils.assertMainThread()
 
+        uiSubscription = delegate.fragmentFlowState().subscribe {
```

Comment:
Since `delegate` uses `BehaviorRelay` for `fragmentFlowState`, we don't need to have the `replay(1)` with `connect` mechanism anymore.